### PR TITLE
Update 02-vvaths.md

### LINKED ISSUE #2: draft issue

### PR DESCRIPTION
**Pull request by vvaths**
fixes #2

### CODE REVIEW

File: _slides/02-vvaths.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (vyas000) [inline _slides/02-vvaths.md:5]: great job!

File: _slides/02-vvaths.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
-Caption here
+Magic Caption!
```
Comment (vyas000) [inline _slides/02-vvaths.md:6]:
```suggestion
Magic Awesome Caption!!!
```

**Review by vyas000 — CHANGES_REQUESTED** [review]
please make the requested update! cool!

File: _slides/02-vvaths.md (hunk 1 above)
Comment (vvaths) [reply _slides/02-vvaths.md:5]: thanks

**Review by vyas000 — APPROVED** [review]
ELGTM!

**Pull request merged**